internal/tools/kops/instance_groups.go

```diff
@@ -83,9 +83,11 @@ func (c *Cmd) UpdateMetadata(metadata *model.KopsMetadata) error {
 			}
 
 			NodeIGCount++
```

Comment:
Due to this line, we are still going to throw the warning + error message below.

Reply:
that is fine, not? at least alert us that we have igs that is not managed by the provisioner